web/router.go

```diff
@@ -175,6 +175,9 @@ func v2Routes(app services.Application, engine *gin.Engine) {
 		ts := TransfersController{app}
 		authv2.POST("/transfers", ts.Create)
 
+		kc := KeysController{app}
+		authv2.POST("/keys", kc.Create)
```

Comment:
I think it'd be worth wrapping this in a check to ensure the CHAINLINK_DEV flag is set here too, or check in the controller method.